lib/ui/form/widget/form_survey_answer_rating_emoji.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_survey/theme/app_colors.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+final _selectedEmojiRatingIndexProvider = StateProvider.autoDispose<int>(
+  (_) => -1,
```

Comment:
I'm thinking about the case if the user proceeds without any selection 💭 

Shall we default it to `3`, then we don't need to think about this case? 🙏